app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,66 @@
+<main>
+  <% if @works.empty? %>
+    <h2> Add some media and start voting! </h2>
+  <% else %>
+    <section class="spotlight">
+      <h2 class="spotlight__header">
+      <span class="spotlight__header--prefix">Media Spotlight: </span>
+      <%= link_to show_spotlight.title, work_path(show_spotlight.id), class:"spotlight__link-to" %><span> by <%= show_spotlight.creator%></span>
+      </h2>
+      <p class="spotlight__description">
+      <%= Vote.where(work_id: show_spotlight.id).count %> - <%= show_spotlight.description %> 
+      </p>
+    </section>
+
+    <hr class="root__hr">
+    <section class="top-ten__container">
+      <section class="top-ten__list_container">
+        <h2 class="top-ten__header">
+        Top Movies
+        </h2>
+        <ul class="list-group top-ten__list">
+          <% Work.top_ten("movie").each do |movie| %>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?